### DISCUSSION TITLE
DIIP V2.1 Specifications

### DISCUSSION THREAD

**Discussion by k00ij** [FIDEScommunity/DIIP#20] [General]
Minor update to DIIP V2.0: OID4VCI WG draft 13 instead of 12 (consensus in the community to not wait until DIIP V3)

OpenID for Verifiable Credential Issuance [WG draft 13](https://openid.net/specs/openid-4-verifiable-credential-issuance-1_0.html)
OpenID for Verifiable Presentations [WG draft 20](https://openid.net/specs/openid-4-verifiable-presentations-1_0.html)
SIOP V2 [WG draft 13](https://openid.net/specs/openid-connect-self-issued-v2-1_0.html)
DIF Presentation Exchange [V2.0](https://identity.foundation/presentation-exchange/spec/v2.0.0/)
W3C Verifiable Credential JWT
DID methods: DID:WEB and DID:JWK
Signature types ES256
Revocation method: StatusList 2021

**Comment by nklomp** [comment]
It is a breaking change, so officially this should become 3.0. Not sure how strict people feel about that
